admin-dev/themes/new-theme/js/components/grid/extension/action/row/submit-row-action-extension.js

```diff
@@ -39,29 +41,62 @@ export default class SubmitRowActionExtension {
       event.preventDefault();
 
       const $button = $(event.currentTarget);
-      const confirmMessage = $button.data('confirm-message');
-
-      if (confirmMessage.length && !window.confirm(confirmMessage)) {
-        return;
-      }
+      const confirmMessage = $button.data('confirmMessage');
+      const confirmTitle = $button.data('title');
 
       const method = $button.data('method');
       const isGetOrPostMethod = ['GET', 'POST'].includes(method);
+      if (confirmTitle !== undefined) {
```

Comment:
What do you want to do there ? The condition should be OK if the value of the data attribute is false, true or an empty string? If not then if(confirmTitle) is enough

Reply:
it's to keep compatibility with not migrated modals

the only difference is that we dont have `data-title` attribute. I think you right if(confirmTitle) would do the job